dependencies_fedora.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+dnf -y install git \
+	make \
+	dbus-devel \
+	gcc \
+	systemd-devel \
+	xfsprogs \
+	python3-pytest \
+	dbus-python \
+	make \
+	device-mapper-persistent-data \
+	python3-coverage \
+	dbus-glib-devel \
+	python3-justbytes.noarch \
+	python3-setuptools \
+	dbus-python-devel \
+	python3-pyudev \
+	python3-dateutil \
+	python3-wcwidth \
+	python3-pyparsing \
+	python3-psutil \
+	dbus-devel.x86_64 \
+	systemd-devel.x86_64 \
+	glibc-devel.x86_64 \
+	dbus-devel.i686 \
+	systemd-devel.i686 \
+	glibc-devel.i686 \
+	openssl-devel \
+	clang \
+	llvm \
+	llvm-devel \
+	keyutils \
```

Comment:
One note is that we should put up an issue to remove this and update the tests to use the Stratis key API once stratis-storage/stratisd#1983 and stratis-storage/stratis-cli#548 are merged.

Reply:
> One note is that we should put up an issue to remove this and update the tests to use the Stratis key API once [stratis-storage/stratisd#1983](https://github.com/stratis-storage/stratisd/pull/1983) and [stratis-storage/stratis-cli#548](https://github.com/stratis-storage/stratis-cli/pull/548) are merged.

@jbaublitz Are you able to do that in the PRS that you cite?

Reply:
I should be able to.

Reply:
> I should be able to.

Ok! That will be the best way.